msteams-platform/bots/how-to/bot-configuration-experience.md

```diff
@@ -54,95 +65,635 @@ You must configure the `fetchTask` property under `bots.configuration` object in
+For more information, see [app manifest schema](../../resources/schema/manifest-schema.md#botsconfiguration).
 
 ### Configure your bot
 
-When a user installs the bot in a team or group chat scope, the `fetchTask` property in the app manifest file initiates `config/fetch` defined in the `teamsBot.js` file. The bot responds with an Adaptive Card and the user provides relevant information in the Adaptive Card and selects **Submit**. After the user selects **Submit**, a `config/submit` is returned to the bot and the bot configuration is complete.
+When a user installs the bot in a team or group chat scope, the `fetchTask` property in the app manifest file initiates `config/fetch` or `config/submit`. The bot responds with an Adaptive Card, when the user provides relevant information in the Adaptive Card and selects Submit, `config/submit` or `config/continue` is returned to the bot and the bot configuration is complete.
+
+You can use the following methods to enable configuration settings for a bot:
+
+1. [Non-Teams method](#non-teams-method).
+
+1. [Teams-specific method](#teams-specific-method).
+
+#### Non-Teams method
 
-You can use `config/fetch` and `config/submit` properties in the `teamsBot.js` file to enable the bot configuration experience in a team or group chat.
+The Non-Teams method simplifies task execution using `OnInvokeActivity` and `OnInvokeActivityAsync`, removing the need for Teams integration:
 
-* Invoke `config/fetch`: You must invoke `config/fetch` task to start the bot configuration flow and define the bot response as required.
+1. `OnInvokeActivity`: The `OnInvokeActivity` method is a fundamental component of the Bot Framework SDK. It enables users to handle incoming activities that aren't recognized by any other method in the bot's logic. This method provides flexibility for handling a wide range of activities, including user interactions, within the bot's conversation flow.
 
-  In the following example, when the bot receives `config/fetch` invoke activity, the bot responds with an Adaptive Card:
+   # [JavaScript](#tab/JS1)
+
+    The `onInvokeActivity` function simplifies the handling of invoke activities. For `config/fetch`, it uses `adaptiveCardForContinue()` to create an Adaptive Card and returns a 200 status response with the card included. For `config/submit`, it checks the choice variable; if its `continue`, it calls `adaptiveCardForSubmit()` for an Adaptive Card and returns a 200 status response with the card. Else, it returns a 200 status response with a completion message.
 
     ```javascript
-    if (context._activity.name == "config/fetch") {
-      const adaptiveCard = CardFactory.adaptiveCard(this.adaptiveCardForDynamicSearch());
-      try {
+    async onInvokeActivity(context) {
+      if (context._activity.name == "config/fetch"){
+        const adaptiveCard = CardFactory.adaptiveCard(this.adaptiveCardForContinue());
```

Comment:
For JS onInvokeActivity, it should contain config/continue and config/auth, here it's still missing config/auth, you can find it our previous discussion in group chat: https://teams.microsoft.com/l/message/19:meeting_YmUwYzNiMjktM2QyZC00MWM3LWIyMDUtYWI1OTIxZGYzOWRl@thread.v2/1709792073568?context=%7B%22contextType%22%3A%22chat%22%7D